releasenotes/notes/prepare-0.24-423de82722d2e31a.yaml

```
@@ -0,0 +1,67 @@
+---
+prelude: |
+  This release is a major feature release that includes numerous new features
+  and bugfixes.
+
+  This release is the final release with support for running Qiskit
+  with Python 3.7. Starting in the next minor version release Python >=3.8 will
+  be required to run Qiskit.
+
+  The highlights of this release:
+
+  QuantumInstance, OpFlow, and algorithms usage deprecation
+  ---------------------------------------------------------
+
+  This release officially deprecates the :class:`~QuantumInstance` class (and
+  its associated helper methods and classes), the :mod:`qiskit.opflow` module,
+  and any usage of those in :mod:`qiskit.algorithms`. This deprecation comes
+  from a long thread of work that started in Qiskit Terra 0.21.0 to refactor
+  the :mod:`qiskit.algorithms` module to be based on the computational
+  :mod:`~qiskit.primitives`. There are associated migration guides for any
```

It might be nice to be consistent with the module names and either truncate all or none 🙂 
```suggestion
  :mod:`qiskit.primitives`. There are associated migration guides for any
```

---

In this case I was kind of cheating. I wanted to say "the computation primitives" but I also wanted primitives to link to the module documentation. So instead of using the ref syntax I did this.

---

Ahh I see! That works of course 👍🏻